packages/binutils.rb

```diff
@@ -70,8 +93,16 @@ def self.check
 
   def self.install
     Dir.chdir 'build' do
+      system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
       system 'make', "DESTDIR=#{CREW_DEST_DIR}", "prefix=#{CREW_PREFIX}",
              "tooldir=#{CREW_PREFIX}", 'install'
     end
   end
+
+  def self.postinstall
+    puts
+    puts 'To use gdb, please install the gdb runtime dependencies with:'.lightblue
+    puts 'crew install gdb'.lightblue
```

Comment:
Wait..what?  The description now says `This also provides the GNU Debugger` but we still need to install gdb?

Reply:
I moved the gdb dependencies into a separate package, so gdb is only really functional with that package, as it installs those dependencies. Should I just not mention gdb in this package description at all?

Reply:
Maybe should remove because it's confusing.

Reply:
irrelevant now due to splitting packages again...